Source/WebKit/WebProcess/Gamepad/WebGamepadProvider.cpp

```diff
@@ -39,6 +39,13 @@
 namespace WebKit {
 using namespace WebCore;
 
+#define WP_MESSAGE_CHECK(assertion, ...) { \
+    if (UNLIKELY(!assertion)) { \
```

Comment:
This needs extra parenthesis around `assertion` (even though you include them below) to be defensive:
```
    if (UNLIKELY(!(assertion))) { \
```